src/layouts/Login/LoginPage.tsx

```diff
@@ -167,6 +166,32 @@ const LoginContent = (): JSX.Element => {
             </Button>
           </TabPanel>
           <TabPanel>
+            <Button
+              onClick={() => getSgidAuth()}
+              isLoading={isSgidAuthLoading}
+              w="full"
+            >
+              Log in with <SingpassLogo mb="-0.2rem" /> app
```

Comment:
could i check why the negative margin is required here?

Reply:
This was because the svg looked slightly off center!